src/main/java/it/gov/pagopa/payhub/activities/exception/InvalidFileException.java

```diff
@@ -0,0 +1,7 @@
+package it.gov.pagopa.payhub.activities.exception;
+
+public class InvalidFileException extends RuntimeException {
```

Comment:
is this exception catched somewhere or it's supposed to be thrown by the exception?
if it's not catched, it should extend Retryable or NotRetryable exception in order to let Temporal to know if the error could be resolved by a retry or not